pkg/kubelet/volume/empty_dir/empty_dir.go

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
I think the argument is you don't put underscores in the directory name either.  I mean, you can totally do it... But then you're the only one.  So.... :)

> On Jan 18, 2015, at 5:35 PM, Tim Hockin <notifications@github.com> wrote:
> 
> In pkg/kubelet/volume/empty_dir/empty_dir.go:
> 
> > @@ -0,0 +1,126 @@
> > +/*
> puke, really? Even for dirs with underscores in their names? Do we have to follow this? I find that sort of obfuscatory
> 
> —
> Reply to this email directly or view it on GitHub.
>